spdb/spatialdb/test/int_test_spatialdb.py

```diff
@@ -464,18 +465,17 @@ def test_cutout_to_black_no_time_single_unaligned_no_iso(self):
         sp.write_cuboid(self.resource, (0, 0, 0), 0, cube1.data)
 
         # write to_black
-        sp.write_cuboid(self.resource, (0, 0, 8), 0, cubeb.data, to_black=True)
+        sp.write_cuboid(self.resource, (0, 0, 0), 0, cubeb.data, to_black=True)
 
         # get cuboid back
         cube2 = sp.cutout(
-            self.resource, (0, 0, 8), (self.x_dim, self.y_dim, self.z_dim), 0)
+            self.resource, (0, 0, 0), (self.x_dim, self.y_dim, self.z_dim), 0)
 
         # expected result
-        cubez = Cube.create_cube(self.resource, [self.x_dim, self.y_dim, self.z_dim//2])
-        cubez.zeros()
-        cubez.morton_id = 0
+        expected_data = cube1.data
+        expected_data[:, :self.z_dim//2, :, :] = 0
```

Comment:
Not a biggie since this happens at the very end of the test, but you are actually mutating `cube1.data` here.